kiva/fonttools/font_manager.py

```diff
@@ -707,14 +710,31 @@ def createFontList(fontfiles, fontext='ttf'):
             except:
                 continue
         else:
+            _, ext = os.path.splitext(fpath)
             try:
-                font = TTFont(str(fpath))
+                if ext.lower() == ".ttc":
+                    collection = TTCollection(str(fpath))
+                    try:
+                        props = []
+                        for font in collection.fonts:
+                            props.append(ttfFontProperty(fpath, font))
+                        fontlist.extend(props)
+                        continue
+                    except Exception:
+                        logger.error(
```

Comment:
A couple of tricks for logging:

- logging does '%' substitution for you automatically(!)
- if you do `exc_info=True` as well it will dump the exception data to the log, which can also be useful.

so you get:
```
logger.error("Could not covert font to FontEntry for file %s", fpath, exc_info=True)
```

Reply:
Sweet! Thanks!